android/src/main/java/com/robinpowered/react/Intercom/IntercomModule.java

```diff
@@ -223,7 +224,30 @@ private UserAttributes convertToUserAttributes(ReadableMap readableMap) {
         Map<String, Object> map = recursivelyDeconstructReadableMap(readableMap);
         UserAttributes.Builder builder = new UserAttributes.Builder();
         for (Map.Entry<String, Object> entry : map.entrySet()) {
-            builder.withCustomAttribute(entry.getKey(), entry.getValue());
+            String key = entry.getKey();
+            Object value = entry.getValue();
```

Comment:
It would be nice if value was checked for the correct type here. Otherwise it may crash if a number is passed for a string param.

Reply:
@ohtangza let me know if you plan to make this change, otherwise I'll merge and publish.

Reply:
Not for now! Bur I will keep track of it, and try to add the logic when I have more time!